<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>380</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>501</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>501</width>
    <height>380</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>501</width>
      <height>241</height>
     </rect>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>481</width>
      <height>24</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Поиск по фамилии</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="findEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushFind">
       <property name="text">
        <string>Найти</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushReset">
       <property name="text">
        <string>Сброс</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>481</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="addData">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeData">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>98</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Выход</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>addData</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_addAbonent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>on_Row(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>362</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableView</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>MainWindow</receiver>
   <slot>slotEditRecord(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushFind</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_findAbonent()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>375</x>
     <y>77</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushReset</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>on_Reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>456</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>41</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_addAbonent()</slot>
  <slot>on_Row(QModelIndex)</slot>
  <slot>slotEditRecord(QModelIndex)</slot>
  <slot>on_findAbonent()</slot>
  <slot>on_Reset()</slot>
 </slots>
</ui>
